stripe/src/main/java/com/stripe/android/paymentsheet/ui/BillingAddressView.kt

```diff
@@ -355,6 +356,17 @@ internal class BillingAddressView @JvmOverloads constructor(
         }
     }
 
+    fun populateFromParams(address: Address?) {
+        address?.let { address ->
```

Comment:
Alternatively, you could do:

```
params.card?.let { cardMultilineWidget.populateFromParams(it) }
```

And then change the argument to be non-null.

Reply:
I cleaned it up so that BaseAddCardFragment doesn't have any let's, so maybe leaving the lets in these lower level functions keeps it looking clean?